tools/build/rules/grpc_c++.bzl

```diff
@@ -110,6 +110,156 @@ def grpc_cc_library(name, srcs = [], public_hdrs = [], hdrs = [],
     alwayslink = alwayslink,
   )
 
+
```

Comment:
@pmuetschard Can you take a look at this part?

Reply:
Done. Only one minor comment, otherwise looks fine.